File: Examples/DotNET/SDK/CSharp/Drawing/GetImageObject.cs
using System;
using Aspose.Cloud;

namespace CSharp.Drawing
{
    class GetImageObject
    {
        public static void Run()
        {
            string subdirectory = "Drawing";

            // The path to the documents directory.
            string dataDir = Common.GetDataDir() + subdirectory + "\\";

            string inputfile = "imagefile.doc";
            string outputfile = "imagefile_out_.png";

            // Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, subdirectory);

            dataDir = dataDir + outputfile;

            // Get the Image Drawing Object from Document
            Common.WordsService.WordsDrawingObjects.ReadDrawingObjectImageData(inputfile, 1, string.Empty, dataDir);

            Console.WriteLine("\nImage object fetched successfully.\nFile saved at " + dataDir);
        }
    }
}

File: Examples/DotNET/SDK/VisualBasic/Drawing/GetImageObject.vb
Imports Aspose.Cloud

Namespace Drawing
    Class GetImageObject
        Public Shared Sub Run()
            Dim subdirectory As String = "Drawing"

            ' The path to the documents directory.
            Dim dataDir As String = (Common.GetDataDir() & subdirectory) + "\"

            Dim inputfile As String = "imagefile.doc"
            Dim outputfile As String = "imagefile_out_.png"

            ' Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, subdirectory)

            dataDir = dataDir & outputfile

            ' Get the Image Drawing Object from Document
            Common.WordsService.WordsDrawingObjects.ReadDrawingObjectImageData(inputfile, 1, String.Empty, dataDir)

            Console.WriteLine(Convert.ToString(vbLf & "Image object fetched successfully." & vbLf & "File saved at ") & dataDir)
        End Sub
    End Class
End Namespace